src/app/constants.ts

```diff
@@ -106,3 +108,7 @@ export function objectToUrlParameters(object: Object) {
     .map((key) => `${key}=${encodeURIComponent(object[key])}`)
     .join('&')
 }
+
+export function buildClientString(requestInfoForm: RequestInfoForm) {
```

Comment:
Please do not duplicate the code we already have on `google-analytics.service`... we can keep the same `this.requestInfoForm || 'Website'` when calling  `reportEvent` since report event already does build the client string when it gets a requestInfoForm

Reply:
🚨I will change this myself since I have to fix the e2e any way